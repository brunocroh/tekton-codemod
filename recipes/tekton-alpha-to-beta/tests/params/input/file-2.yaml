spec:
  inputs:
    params:
      - name: ADDR
        value: https://example.com/foo.json
