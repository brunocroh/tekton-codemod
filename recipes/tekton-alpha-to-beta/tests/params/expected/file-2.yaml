spec:
  params:
    - name: ADDR
      value: https://example.com/foo.json
